### PR TITLE
Bugfix/non timestamp sentinel value

### PR DESCRIPTION
**Pull request by sas1ni69**
Fixes https://github.com/rubysherpas/paranoia/pull/531

### CODE REVIEW

**Comment by mathieujobin** [comment]
@sas1ni69 You could have done `git cherry-pick 36207efd0d45`

now with your current branch, you could do `git rebase -i origin/core` and in the editor that opens, you enter as the first line

`pick  36207efd0d45`

the second pick line is equivalent, you could remove it, or it should disappear automatically, if your commit is `100% 1:1`

or I might just merge his PR first.. 

Thanks for the quick fix.

File: lib/paranoia.rb [hunk 1]
```diff
@@ -296,12 +296,16 @@ def paranoia_column
...
+      send(paranoia_column)
+    end
+
     def paranoia_sentinel_value
       self.class.paranoia_sentinel_value
     end
 
     def deletion_time
-      send(paranoia_column)
+      paranoia_column_value.acts_like?(:time) ? paranoia_column_value : deleted_at
```
Comment (mathieujobin) [inline lib/paranoia.rb:308]: interesting

**Comment by sas1ni69** [comment]
@mathieujobin much better! Thanks for merging his PR.

I tried to cherry pick but got bad object. I think it was because I hadn't set up upstream.

**Comment by mathieujobin** [comment]
@sas1ni69 I have a git command that makes this easy available in https://github.com/mathieujobin/git-scripts

I only have to do `git fork-checkout sas1ni69:bugfix/non_timestamp_sentinel_value` to checkout this PR locally.
the last argument being a quick copy from the top of this page.

**Comment by sas1ni69** [comment]
That looks awesome! Really appreciate you sharing, gonna pick up some of your commands 👯

**Comment by mathieujobin** [comment]
I hope you find many useful ;) 

`git xrebase_all_branches` might appear scary, but I use it a lot 

`git amendforcepush` is very useful for quick typo you found after pushing something, is use it a lot part of my workflow as well

and `git newbranch` but its got `origin/develop` hardcoded as the parent for now

**Pull request merged**